code/datums/outfits/outfit_debug.dm

```diff
@@ -16,13 +16,14 @@
 	mask = /obj/item/clothing/mask/gas/welding/advanced
 	shoes = /obj/item/clothing/shoes/combat/swat
 
+	// shit down here is mine
```

Comment:
??????

Reply:
I already removed in a more recent PR, that should be removed when merging master